src/components/shared/WithSize.js

```diff
@@ -53,7 +54,7 @@ export function withSize<
         throw new Error('Unable to find the DOMNode');
       }
       this._container = container;
-      window.addEventListener('resize', this._resizeListener);
+      getResizeObserverWrapper().subscribe(container, this._resizeListener);
```

Comment:
Hm, so previously we were listening the resize in the whole window? It's nice that we are listening for the container now.